app/javascript/application.js

```diff
@@ -1,3 +1,4 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "controllers"
 import "@hotwired/turbo-rails"
+import "bootstrap"
```

Comment:
Bootstrap _does_ have an es-module compatible distribution, too. But, not distributed with the Gem. Maybe it would be better to disconnect the two? It would be the more "modern" way to do JS, at the expense of having to keep the two different Bootstrap packages in sync.

Reply:
Same-source seems much more important to me. If we did want to get the ESM version (which, in my experience, only affects how you import it), I'd vote for either vendoring the whole thing (css included), or `npm`ing it.